aexpect/session_ops.py

```diff
@@ -0,0 +1,598 @@
+# This Python file uses the following encoding: utf-8
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+
+"""
+
+SYNOPSIS
+------------------------------------------------------
```

Comment:
We could have a more markdown/rst here, but I would recommend to reduce the number of `-` to the title lenghts.

Reply:
+1